player.py

```diff
@@ -40,6 +41,10 @@ def set_position(self, position):
 
     def render(self):
         centre = self.map.get_centre()
+        font = pygame.font.Font(None, 30)
+        font.set_bold(True)
+        box = font.render(self.name, False, (255, 255, 255))
+        self.screen.blit(box, (centre[0], centre[1] - 20)) # Draws name above player
```

Comment:
Only your player will be in the centre of the screen. Other players will be offset by the `offset_centre` coordinates defined on line 52. Should just be a small tweak to render their names relative to their position.